packaging/nuget/package.ps1

```diff
@@ -31,4 +31,4 @@ foreach ($ProjectName in $ProjectsToPack) {
     }
 }
 
-Get-ChildItem $IntermediatePackagesDir\$Configuration -Filter *.nupkg | ? {$_.Name -NotLike "*.symbols.nupkg"} | Copy-Item -Destination $PackagesDir
+Get-ChildItem $IntermediatePackagesDir -Filter *.nupkg | ? {$_.Name -NotLike "*.symbols.nupkg"} | Copy-Item -Destination $PackagesDir
```

Comment:
That is what we decided with design -o always outputs flat result into directory. I think incremental compilation doesn't take nupkg's into account.